pkg/kwok/server/debugging_logs.go

```diff
@@ -47,7 +47,11 @@ func (s *Server) GetContainerLogs(ctx context.Context, podName, podNamespace, co
 	}
 
 	opts := crilogs.NewLogOptions(logOptions, time.Now())
-	return readLogs(ctx, log.LogsFile, opts, stdout, stderr)
+	logsFile := log.LogsFile
+	if logOptions.Previous {
+		logsFile = log.PreviousLogsFile
+	}
+	return readLogs(ctx, logsFile, opts, stdout, stderr)
```

Comment:
There may be a few more cases to consider
- The final logsFile is empty
- Both --follow and --previous are enabled at the same time